f2/src/utils/formatAnalystEmail.js

```diff
@@ -64,10 +64,17 @@ const formatVictimDetails = (data) => {
 
   let postalCity = ''
   let postalProv = ''
-
-  if (data.location.postalCode) {
-    postalCity = zipcodes.lookup(data.location.postalCode).city
-    postalProv = zipcodes.lookup(data.location.postalCode).state
+  try {
+    let location = zipcodes.lookup(data.location.postalCode)
+    if (location === undefined) {
+      postalCity = 'Location lookup is not found'
+      postalProv = 'Location lookup is not found'
+    } else {
+      postalCity = zipcodes.lookup(data.location.postalCode).city
+      postalProv = zipcodes.lookup(data.location.postalCode).state
+    }
```

Comment:
postalCity = location.city
   postalProv = location.state

Reply:
+1

Reply:
fixed

Reply:
Just testing the variable hoisting and will confirm if the 2 vars assigned inside the catch statement will be re-assigned to the error string attached, just recalling any assignment for vars inside catch statement remain local in the scope of this block only

Reply:
https://medium.com/@sarbbottam/the-curious-case-of-variable-declarations-inside-a-catch-block-891a4369aaf0

Reply:
Just need to be sure that our catch statement string assignments of this error msg will be assigned successfully to the scope of these 2 vars in case of failure

Reply:
@ngosset can we test one failure postal code in the email and check if the data received will indeed have Location lookup is not found ` lookup is not found` or `Location lookup failed` i'm doubting that `Location lookup failed` will be assigned in case of errors

Reply:
an example is listed here 
<img width="821" alt="Screen Shot 2020-06-15 at 12 23 39 PM" src="https://user-images.githubusercontent.com/2603117/84682063-16c51980-af03-11ea-9ccc-8f7465555523.png">